bridge-proxy/src/config.rs

```diff
@@ -23,8 +23,11 @@ pub trait ConfigModule {
     }
 
     #[only_owner]
-    #[endpoint(setBridgedTokensWrapper)]
-    fn set_bridged_tokens_wrapper(&self, opt_address: OptionalValue<ManagedAddress>) {
+    #[endpoint(setBridgedTokensWrapperAddress)]
+    fn set_bridged_tokens_wrapper_contract_address(
```

Comment:
Will need to add this step in the setup process
This should also be added in the multisig/interaction/config scripts.